selfdrive/car/hyundai/interface.py

```diff
@@ -279,6 +279,15 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[], disabl
       ret.steerRatio = 12.069
       ret.lateralTuning.pid.kiBP, ret.lateralTuning.pid.kpBP = [[0.], [0.]]
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.16], [0.01]]
+    elif candidate == CAR.GENESIS_GV70:
+      ret.mass = 1513. + STD_CARGO_KG
+      ret.wheelbase = 2.84
+      ret.steerRatio = 13.27 * 1.15   # 15% higher at the center seems reasonable
+      ret.safetyConfigs = [get_safety_config(car.CarParams.SafetyModel.noOutput),
+                      get_safety_config(car.CarParams.SafetyModel.hyundaiHDA2)]
```

Comment:
Let's make this use HDA2_CAR

Reply:
@sshane  What's the best way to configure the configure_torque_tune settings? I see a bunch of docs about pids/indi, but not much on the torque tune. Would like to stick with the new method/what the ev6 is using.

Reply:
I should write something about that, basically just set your override like you did and just add that `configure_torque_tune` line to your new platform